netlogo-gui/src/main/headless/HeadlessWorkspace.scala

```diff
@@ -117,7 +117,9 @@ with org.nlogo.api.ViewSettings {
   /**
    * Has a model been opened in this workspace?
    */
-  var modelOpened = false
+  private[this] var _openModel = Option.empty[Model]
+  def modelOpened = _openModel.nonEmpty
+  def setModelOpened(model: Model): Unit = { _openModel = Some(model) }
```

Comment:
Shouldn't it be `setOpenModel`?

(The comment above is not consistent with the code anymore)

Reply:
That's a better name. I can also rearrange is so that `modelOpened` directly follows the comment.